kernel/enter.c

```diff
@@ -203,6 +203,30 @@ static int _setup_ext2(const char* rootfs)
 }
 #endif /* MYST_ENABLE_EXT2FS */
 
+#if defined(MYST_ENABLE_HOSTFS) && defined(MYST_DEBUG)
+static int _setup_hostfs(const char* rootfs)
+{
+    int ret = 0;
+
+    if (myst_init_hostfs(&_fs) != 0)
+    {
+        myst_eprintf("failed initialize the RAM files system\n");
```

Comment:
Should the message be `failed to initialize the host files system`?

Reply:
Thanks. Fixed.